src/blocks/weather.rs

```diff
@@ -32,13 +33,16 @@
 //! `coordinates` | GPS latitude longitude coordinates as a tuple, example: `["39.2362","9.3317"]` | Yes* | None
 //! `units` | Either `"metric"` or `"imperial"`. | No | `"metric"`
 //! `lang` | Language code. See [here](https://openweathermap.org/current#multi). Currently only affects `weather_verbose` key. | No | `"en"`
+//! `forcast_hours` | How many hours should be forcast (must be increments of 3 hours, max 120 hours) | No | 0
 //!
 //! One of `city_id`, `place` or `coordinates` is required. If more than one are supplied, `city_id` takes precedence over `place` which takes place over `coordinates`.
 //!
 //! The options `api_key`, `city_id`, `place` can be omitted from configuration,
 //! in which case they must be provided in the environment variables
 //! `OPENWEATHERMAP_API_KEY`, `OPENWEATHERMAP_CITY_ID`, `OPENWEATHERMAP_PLACE`.
 //!
+//! Forcasts are disabled by default as they consume another api call
```

Comment:
I'm not sure that it's really a big deal to make this extra call... What do you think?

Reply:
I think it is a good idea to avoid unnecessary API calls. However we can automatically enable the forecast if the format template contains any templates that rely on it.